src/infinite-list/index.html

```diff
@@ -1,17 +1,17 @@
 <ul ref="wrapper" class="gb-list__wrapper {isGrid ? 'gb-list__wrapper--grid' : 'gb-list__wrapper--column'}">
-  <li data-is="gb-list-item" class="load first-loader" if="{$infinite.isFetchingBackward}" _consumes="infinite">
-    <gb-infinite-loader label="{$infinite.loaderLabel}"></gb-infinite-loader>
+  <li data-is="gb-list-item" class="load first-loader" _consumes="infiniteList" if="{props.isFetchingBackward}" _props="{parent.childProps()}">
+    <gb-infinite-loader label="{$infiniteList.loaderLabel}"></gb-infinite-loader>
   </li>
-  <li class="load" if="{$infinite.loadMore && $infinite.prevExists}">
-    <gb-button class="load-prev" on-click="{$infinite.clickPrev}">Load Previous</gb-button>
+  <li data-is="gb-list-item" class="load" _consumes="infiniteList" if="{props.loadMore && props.prevExists}" _props="{parent.childProps()}">
+    <gb-button class="load-prev" on-click="{$infiniteList.clickPrev}">Load Previous</gb-button>
   </li>
-  <li data-is="gb-list-item" class="gb-list__item" each="{item, i in props.items}">
+  <li data-is="gb-list-item" class="gb-list__item" each="{item, i in props.items}" _props="{parent.childProps()}">
```

Comment:
**keep this one as `parent.childProps()`** (because it's in an each it's in a child context)